rust_src/src/frames.rs

```diff
@@ -523,13 +523,13 @@ pub fn next_frame(frame: LispFrameOrSelected, miniframe: LispObject) -> LispFram
     // a valid candidate will be returned regardless of its position.
     while passed < 2 {
         for_each_frame!(f => {
-            if passed > 0 {
-                let tmp = unsafe { candidate_frame(f.into(), frame_obj, miniframe) };
-                if !tmp.is_nil() {
+        if passed > 0 {
```

Comment:
Are these changes what you intended? It seems odd for me :confused:

I ran `rustfmt` for original code and changed one, and for both codes `rustfmt` changed nothing. I'm wondering why.

Reply:
Hmm. It seems to be the right version since the travis build doesn't fail.

Did you remove and add rustfmt-preview after we updated the toolchain ?

Reply:
@brotzeit I think the error is on your side. I resolved this spacing issue when I updated the compiler version.

Reply:
I don't get it...